src-plugins/medVtkView/medVtkView.cpp

```diff
@@ -47,6 +47,7 @@ PURPOSE.
 #include <medParameterPoolManager.h>
 #include <medSettingsManager.h>
 #include <medClutEditorToolBox/medClutEditorToolBox.h>
+#include <medGlobal.h>
```

Comment:
I do not understand why naming this file "medGlobal", do you want to add in it some functions as in medUtilities? Can it be included in medUtilities?